test/integration/noderesourcetopology_test.go

```diff
@@ -52,6 +52,10 @@ const (
 	memory = string(v1.ResourceMemory)
 )
 
+const (
```

Comment:
the commit message says `negative unit-tests` but we are changing the integration tests - let's make it consistent